README.md

```diff
@@ -0,0 +1,43 @@
+# CAOTIC
+High Contrast and Adaptive Optics laboratory testbeds community website
+
+Welcome to the community website for astronomical testbeds! We are currently working on getting our 
+website migrated to highconaotools.github.io, the original website can be found here:  
+https://sites.google.com/view/highcontrastlabs/
+
+## Local setup - MacOS
+
+### Initial setup
+
+If you use conda, make sure you have deactivated all your environments (including `base`) by doing `$ conda deactivate` often enough.
+
+*On a Mac*:
+1. Install homebrew following the instructions on this page:  
+https://osxdaily.com/2018/03/07/how-install-homebrew-mac-os/
+
+2. Install the package manager rbenv (more on rbenv [here](https://github.com/rbenv/rbenv) ) 
+`$ brew install rbenv`
+
+3. Set up rbenv in your shell  
+`$ rbenv init`
+
+4. Close and reopen your terminal for the changes to take effect
+
+5. Install ruby (more about using rbenv on a Mac [here](https://medium.com/@norton.seanm/a-guide-to-using-rbenv-mac-4ca211b1c713))  
+`$ rbenv install 2.7.2`
+
+6. Set your local ruby version to use  
+`$ rbenv local 2.7.2`
+
+7. Install bundler and jekyll  
+`$ gem install bundler jekyll`
+
+[Jekyll installation on Mac](https://jekyllrb.com/docs/installation/macos/)  
+[Jekyll general installation docs](https://jekyllrb.com/docs/installation/)
+
+
+### Render the website locally
+
+1. cd into the repository
+2. check out the branch that you want to build locally
+3. `$ jekyll serve`
```

Comment:
After running `jeyll serve` this is all that happens: 
![Screen Shot 2021-04-06 at 4 49 43 PM](https://user-images.githubusercontent.com/17128221/113776664-24e56000-96f8-11eb-88b4-e7b88ca8be9b.png)

I normally would expect the page to open automatically, but that doesn't happen. Is there an additional step missing?

Reply:
Ah, I just noticed that if I copy the server address into my address bar, I see the website. Is that the final step?

Reply:
Yes - I was wondering myself why it doesn't open automatically for me either, this might have something to do with local browser settings. I will add a lite that tells people to copy paste the server address into their browser in order to open the site (in your case `http://127.0.0.1:4000/`)

Reply:
I advise you to run the command "jekyll serve --livereload". It will refresh your local website every time you change something in the code.

Reply:
Does that also work when you don't have any website currently open?

Reply:
Sure !
I run this command every time, then in the terminal (I have what we can see in the Keira's screen shot) i find the link to the local website on which I Ctrl+click to open it in a new tab.